### ISSUE TITLE
Document tab context menu not reachable via keyboard

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#8851]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.4.1532
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
* open a file in the source pane
* right-click on the tab, notice the various commands (Close, Close All, etc.)
* close the menu
* try to invoke this menu entirely via keyboard

### Describe the problem in detail
* can't do it

### Describe the behavior you expected
* should be a way to use this menu without a pointing device

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Initially set for juliet-rose. I need to ponder a bit, could fairly easy make the case that a keyboard user could do these things using the existing menu commands and/or the command-palette.

Except the new "Copy Path" command, that's currently only available on the context menu.

**Comment by mikebessuille** [comment]
We think this is not urgently necessary as all the functions within the context menu are accessible via the keyboard, just not the menu itself.

**Comment by gtritchie** [comment]
Closing; these commands are available to keyboard-only users via the File menu, keyboard shortcuts, and the command palette. It isn't necessary to also make them available via the tab context menu.
